[2022-10-17 10:30:12,111] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.store_user backfill__2022-09-10T00:00:00+00:00 [queued]>
[2022-10-17 10:30:12,119] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.store_user backfill__2022-09-10T00:00:00+00:00 [queued]>
[2022-10-17 10:30:12,119] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:30:12,119] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-17 10:30:12,120] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-17 10:30:12,128] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_user> on 2022-09-10 00:00:00+00:00
[2022-10-17 10:30:12,134] {standard_task_runner.py:52} INFO - Started process 9641 to run task
[2022-10-17 10:30:12,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'store_user', 'backfill__2022-09-10T00:00:00+00:00', '--job-id', '95', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp6n9elqc7', '--error-file', '/tmp/tmp3w0hhfe3']
[2022-10-17 10:30:12,137] {standard_task_runner.py:80} INFO - Job 95: Subtask store_user
[2022-10-17 10:30:12,333] {task_command.py:369} INFO - Running <TaskInstance: my_first_dag.store_user backfill__2022-09-10T00:00:00+00:00 [running]> on host edecb506ad75
[2022-10-17 10:30:12,375] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-09-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-09-10T00:00:00+00:00
[2022-10-17 10:30:12,375] {postgres.py:127} INFO - Running copy expert: COPY users2 FROM stdin WITH DELIMITER AS ',', filename: /tmp/processed_user.csv
[2022-10-17 10:30:12,380] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-17 10:30:12,385] {python.py:173} INFO - Done. Returned value was: None
[2022-10-17 10:30:12,393] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=store_user, execution_date=20220910T000000, start_date=20221017T103012, end_date=20221017T103012
[2022-10-17 10:30:12,434] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-17 10:30:12,545] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
